model/host/hostutil.go

```diff
@@ -230,16 +230,22 @@ func (h *Host) jasperExtractedFileName(config evergreen.JasperConfig) string {
 // BootstrapScript creates the user data script to bootstrap the host.
 func (h *Host) BootstrapScript(config evergreen.JasperConfig) string {
 	if h.Distro.IsWindows() {
-		cmds := h.FetchJasperCommandWithPath(config, "/bin")
+		cmds := []string{
+			h.FetchJasperCommandWithPath(config, "/bin"),
+			h.ForceReinstallJasperCommand(config),
+		}
 		// PowerShell nested quotation marks are handled by using two quotation
 		// marks.
-		quotedCmds := strings.Replace(cmds, "'", "''", -1)
+		quotedCmds := make([]string, 0, len(cmds))
+		for _, cmd := range cmds {
+			quotedCmds = append(quotedCmds, strings.Replace(cmd, "'", "''", -1))
+		}
 		commands := []string{
 			"<powershell>",
 			fmt.Sprintf("%s -c '%s'", h.Distro.ShellPath, quotedCmds),
 			"</powershell>",
 		}
 		return strings.Join(commands, "\r\n")
```

Comment:
should \r\n be in a constant? or is it a constant?

Reply:
There's no constant in the standard library for this. All the stdlib code that I've seen seems to use "\r\n" or define an unexported const.